### PR TITLE
complete GET /api/articles/:article_id/comments endpoint

### PR DESCRIPTION
**Pull request by zajk9999**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -103,4 +103,57 @@ describe("/api/articles", () => {
...
+})
+
+
+describe("/api/articles/:article_id/comments", () => {
+	it("GET: 200 - responds with a correct comments array", () => {
+		return request(app)
+			.get("/api/articles/3/comments")
+			.expect(200)
+			.then(({ body }) => {
+				body.comments.forEach((comment) => {
```
Comment (nc-haz-mallion) [inline __tests__/app.test.js:115]: Technically this test could provide a false positive as the forEach will not execute if an empty array is returned. 
Please also add an expect statement that checks the length of the comments array to prevent a false positive

File: __tests__/app.test.js [hunk 2]
```diff
@@ -103,4 +103,57 @@ describe("/api/articles", () => {
...
+	})
+	it("GET: 200 - responds with an array of comments in the descending order", () => {
+		return request(app)
+			.get("/api/articles/3/comments")
+			.expect(200)
+			.then(({ body }) => {
+				expect(body.comments).toBeSortedBy("created_at", {descending: true});
+			});
+	});
+	it("GET: 200 - responds with en empty arrat when given an id for the existing article, but there is no comments", () => {
```
Comment (nc-haz-mallion) [inline __tests__/app.test.js:134]: Very nice edge case to check for -
 also there's a slight typo in your test description here

**Review by nc-haz-mallion — COMMENTED** [review]
Some really nice code here, easy to read and clear naming conventions throughout, well done!
